events/events.go

```diff
@@ -189,15 +190,28 @@ func reconcileResponderEvent(tx *gorm.DB, event api.Event) error {
 	ctx := api.NewContext(tx)
 
 	var _responder api.Responder
-	err := tx.Where("id = ? AND external_id is NULL", responderID).Preload("Team").Find(&_responder).Error
+	err := tx.Where("id = ? AND external_id is NULL", responderID).Preload("Incident").Preload("Team").Find(&_responder).Error
 	if err != nil {
 		return err
 	}
 
+	teamSpec, err := _responder.Team.GetSpec()
+	if err != nil {
+		return err
+	}
+
+	shoutrrr, err := notification.NewShoutrrrClient(ctx, teamSpec.Notifications)
```

Comment:
I would loop over each notification, and then create a client for it, not create the intermediary objects to contain multiple objects and multiple errors. 

I think we would also need to loop over the notifications and then emit new events (`notification.<type>`), 1 per notification (if it filters) so that if 1 notification goes through, it doesn't get retried with another that fails

Reply:
> I would loop over each notification, and then create a client for it, not create the intermediary objects to contain multiple objects and multiple errors.

Our Shoutrrr client internally has a separate service - one for each notification. So, we can handle each notification in a case by case basic just like if we would have multiple shoutrrr clients.